digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/feedback/FeedbackDao.scala

```diff
@@ -0,0 +1,75 @@
+package fi.liikennevirasto.digiroad2.dao.feedback
+
+import slick.driver.JdbcDriver.backend.Database
+import Database.dynamicSession
+import com.github.tototoshi.slick.MySQLJodaSupport._
+import fi.liikennevirasto.digiroad2.oracle.MassQuery
+import slick.jdbc.StaticQuery.interpolation
+import org.joda.time.DateTime
+import fi.liikennevirasto.digiroad2.service.feedback.FeedbackInfo
+import slick.jdbc.{GetResult, PositionedResult, StaticQuery}
+
+class FeedbackDao {
+
+
+  def getApplicationFeedbackByStatus(status: Boolean): Seq[FeedbackInfo] = {
+    val feedbackFilter = if (status) " status = 1 " else " status = 0 "
+    val query =
+      s"""
+        select id, receiver, created_by, created_date, subject, body, status, status_date
+        from feedback
+        where $feedbackFilter
+        """
+    StaticQuery.queryNA[FeedbackInfo](query).iterator.toSeq
+
+  }
+
+  implicit val getFeedback = new GetResult[FeedbackInfo] {
+    def apply(r: PositionedResult) = {
+      val id = r.nextLong()
+      val receiver = r.nextStringOption()
+      val createdBy = r.nextStringOption()
+      val createdAt = r.nextTimestampOption().map(timestamp => new DateTime(timestamp))
+      val subject = r.nextStringOption()
+      val body = r.nextStringOption()
+      val status = r.nextBoolean()
+      val statusDate = r.nextTimestampOption().map(timestamp => new DateTime(timestamp))
+
+      FeedbackInfo(id, receiver, createdBy, createdAt, body, subject, status, statusDate)
+    }
+  }
+
+  def getAllFeedbacks(): Seq[FeedbackInfo] = {
+    val query =  s"""
+            select id, receiver, created_by, created_date, subject, body, status, status_date
+            from feedback
+            """
+      StaticQuery.queryNA[FeedbackInfo](query).iterator.toSeq
```

Comment:
Since we are using the same select query 3 times why we don't create a getAllFeedbacks with an optional "where" input parameter.
So, each time we want using we call the getAllFeedbacks with different "where" condition.